lib/build.js

```diff
@@ -21,6 +21,7 @@ exports.usage = 'Invokes `' + (win ? 'msbuild' : 'make') + '` and builds the mod
 function build (gyp, argv, callback) {
   var release = processRelease(argv, gyp, process.version, process.release)
     , makeCommand = gyp.opts.make || process.env.MAKE
+      || (process.platform.indexOf('aix') != -1 && 'gmake')
```

Comment:
the mixed use of `&&` and the ternary operator below makes this a bit confusing, the duplicate resolving of `gmake` suggests redundancy, and, shouldn't it just be `aix` when it's AIX? the `indexOf` is for the *bsd variants I _think_.

Can I suggest squashing both these lines to: `((/(^aix$)|bsd/).test(process.platform) ? 'gmake' : 'make')`